Snowdog_Components/components/Templates/product-view/_product-view.scss

```diff
@@ -731,4 +731,8 @@
             bottom: $product-view__grecaptcha-badge-bottom\@medium !important;
         }
     }
+
+    .paypal-express-checkout {
+        z-index: 0;
+    }
```

Comment:
please move it to theme styles and create variable based on global z-index variable